onnx/defs/tensor/defs.cc

```diff
@@ -2473,7 +2473,7 @@ ONNX_OPERATOR_SET_SCHEMA(
 
 ONNX_OPERATOR_SET_SCHEMA(
     Identity,
-    14,
+    16,
```

Comment:
why is this change part of this PR? bad merge?

Reply:
Yes.. reverted